### DISCUSSION TITLE
Help with ECG Viewer & does it handle fused imaging like PETCT

### DISCUSSION THREAD

**Discussion by Vik1922** [nroduit/Weasis#367] [General]
Using Windows 11 OS
Installed weasis from the Microsoft store.

Q1 -ECG
How to use ECG viewing tool in Weasis?
What should be the 'input data set' -JPG or a proprietary data set, please specify what type of files it accepts.
could not find a ECG option in the menu.
if it is a separate plug in how do i install it

Q2
Can weasis fuse & display PET-CT images?

also can it do an MPR on a NM Tomo series? tried but it did not work

thanks in advance.

**Comment by nroduit** [comment]
1) Weasis is a DICOM viewer, so you need to have DICOM ECG. Proprietary formats are not supported.

2) Fuse of PET-CT is not supported yet. MPR should work in any series which contains 3d position and orientation, please provide an example.
